plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation/DraggableAnnotationController.java

```diff
@@ -62,11 +63,15 @@ void onSourceUpdated() {
     stopDragging(draggedAnnotation);
   }
 
-  void onMoveBegin(MoveGestureDetector detector) {
+  boolean onMoveBegin(MoveGestureDetector detector) {
     if (detector.getPointersCount() == 1) {
       T annotation = annotationManager.queryMapForFeatures(detector.getFocalPoint());
-      startDragging(annotation);
+      if (annotation != null) {
+        startDragging(annotation);
```

Comment:
I think `startDragging` should return a `boolean` as well because we still want to return `false` if the annotation is not draggable, or we can move that check up.

Reply:
I thought about making `startDragging` `stopDragging` private as they are only used in tests